miso-web/src/main/webapp/scripts/box_ajax.js

```diff
@@ -311,6 +313,88 @@ Box.ui = {
       filter = '^(?:' + positionStrings.join('|') + ')$';
     }
     t.fnFilter(filter, 0, true);
+    Box.ui.getBulkActions(positionStrings);
+  },
+  
+  getBulkActions: function(positionStrings) {
+    function addToolbarMemo(memo) {
+      // empty all previous bulk actions and/or info
+      Box.ui.createToolbarIfNecessary();
+      var toolbar = jQuery('#listingBoxablesToolbar');
+      toolbar.empty();
+      jQuery('<span/>', {
+          'text': memo
+      }).appendTo(toolbar);
+    }
+    
+    var items;
+    if (positionStrings) {
+      items = Box.visual.data.filter(function(item) { return positionStrings.indexOf(item.coordinates) >= 0 });
+    } else {
+      // if no items are selected, determine bulk actions based on the entity types of all box contents
+      items = Box.visual.data;
+    }
+    if (items.length < 1) {
+      // empty box
+      addToolbarMemo("Add items to box to see bulk actions.");
```

Comment:
This shows up after selecting and deselecting items, and when selecting empty positions only too